lib/gitter-adapter.js

```diff
@@ -329,9 +329,9 @@ Adapter.prototype.messageOfTheDay = function() {
   var c = this.client;
 
   c.send(c.host, irc.reply.motdStart, c.nick, ':- Message of the Day -');
-  c.send(c.host, irc.reply.motd,      c.nick, 'Welcome To Gitter IRC!');
-  c.send(c.host, irc.reply.motd,      c.nick, 'Info at https://irc.gitter.im');
-  c.send(c.host, irc.reply.motd,      c.nick, 'Code at https://github.com/gitterHQ/irc-bridge');
+  c.send(c.host, irc.reply.motd,      c.nick, ':Welcome To Gitter IRC!');
```

Comment:
According to that spec, wouldn't it be prefixed with `:- ` ?

I am looking at 372 `RPL_MOTD`